packages/theme/src/tokens/zIndices.ts

```diff
@@ -0,0 +1,91 @@
+export const zIndices = {
```

Comment:
```suggestion
export const zIndexes = {
```

Reply:
We need to follow theme ui spec https://theme-ui.com/theme-spec

Reply:
Ok!